CMakeLists.txt

```
@@ -167,6 +172,7 @@ set_target_properties(WarpX PROPERTIES
 
 # link dependencies
 target_link_libraries(WarpX PUBLIC WarpX::thirdparty::AMReX)
+target_link_libraries(WarpX PUBLIC pxrmp_qed)
```

With the alias proposed in PICSAR:
```suggestion
target_link_libraries(WarpX PUBLIC pxrmp_qed::pxrmp_qed)
```